lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1280,6 +1237,77 @@
   }
 }
 
+// view - discussions management
+// --------------------
+.instructor-dashboard-wrapper-2 section.idash-section#discussions_management {
+
+  .form-submit {
+    @include idashbutton($uxpl-blue-base);
+    @include font-size(14);
+    @include line-height(14);
+    margin-right: ($baseline/2);
```

Comment:
I was under the understanding that the scope of this story was to move the existing UI, should I also be updating the existing UI?  I do not think we scoped that in this story.

The SASS as of now is the existing SASS without me altering anything.  I can go through that process, but this PR will not merge for a few more days.